build.sbt

```diff
@@ -67,15 +67,23 @@ def isolateAllTests(tests: Seq[TestDefinition]) = tests map { test =>
   } toSeq
 
 // Subproject definitions begin
+//
```

Comment:
We need to keep FIRRTL unmanaged. IIRC it is because RC also has it unmanaged.

Reply:
chisel-testers seems to break if we don't add firrtl as a dependency for it, though

Reply:
Oh nvm, I was reading that wrong. Seems like the problem is just that I didn't give the right dependencies to sha3.

Reply:
Nope, I was right the first time. RocketChip doesn't have a dependency on firrtl in its build.sbt, but firrtl-interpreter, chisel-testers, and treadle do, so we will have to manage a dependency for them it seems.